zuul.d/projects.yaml

```diff
@@ -21,24 +21,20 @@
     name: ansible/ansible
     periodic-0/12:
       jobs:
-        - openstacksdk-ansible-functional-devstack:
+        - openstacksdk-ansible-v2.8.0-functional-devstack-stein:
             branches: devel
-        - openstacksdk-ansible-functional-devstack-stein:
+        - openstacksdk-ansible-v2.8.0-functional-devstack-rocky:
             branches: devel
-        - openstacksdk-ansible-v2.8.0-functional-devstack-stein:
+        - openstacksdk-ansible-v2.8.0-functional-devstack-queens:
+            branches: devel
```

Comment:
maybe use `branches: stable-2.8`, looks more easy to understand

Reply:
I think clear tag version is better, because it's regular release version and have been applied in `pip`, `yum` and `apt`, `stable-2.8` maybe cause some confusions, like: `2.8.0` or `2.8.1`?

```
[root@arm-chenrui spark]# pip search ansible | grep "ansible (2.8.0)"
ansible (2.8.0)                            - Radically simple IT automation
```

Reply:
I use the release name of ansible,  and I think we should give the whole name.